optuna/storages/_rdb/storage.py

```diff
@@ -283,8 +291,11 @@ def get_study_direction(self, study_id: int) -> StudyDirection:
 
         with _create_scoped_session(self.scoped_session) as session:
             study = models.StudyModel.find_or_raise_by_id(study_id, session)
+            direction = tuple(
+                [d.direction for d in models.StudyDirectionModel.where_study(study, session)]
+            )[0]
```

Comment:
This is a milestone for the next MO integration. As soon as this PR is merged, we will rename `get_study_direction` as `get_study_directions` and return the output as a tuple, making it multidimensional.

Reply:
I chose `tuple` instead of `list` to be consistent with the existing multi-objective module. In the existing MO module, `MultiObjectiveFrozenTrial.values` is a `tuple` but `MultiObjectiveStudy.directions` is a `list`. After discussing the integration of the MO module with @sile, it's better to unify with a `tuple`. But if `list` is more consistent with the other storage functions, then maybe we should choose `list` for storage. What do you think?

Reply:
I think either `list` or `tuple` is okay if it's consistent with the other part of the Optuna code base.

Reply:
I will change to use `list` instead of `tuple` for consistency with other storage functions. In PR #1994, to align with the storage functions, how about making the return types of `Study.directions` and `FrozenTrial.values` will be `list`.?